src/app/Main.js

```diff
@@ -98,9 +98,10 @@ function runApp(initial_state) {
 
 if (!window.Intl) {
     require.ensure(['intl/dist/Intl'], (require) => {
-        window.IntlPolyfill = window.Intl = require('intl/dist/Intl')
-        require('intl/locale-data/jsonp/en-US.js')
-        require('intl/locale-data/jsonp/es.js')
+        window.IntlPolyfill = window.Intl = require('intl/dist/Intl');
```

Comment:
these changes should be removed from this PR if we're no longer addressing internationalization here